backend/infrahub/task_manager/constants.py

```diff
@@ -0,0 +1,15 @@
+from infrahub.core.constants import TaskConclusion
+
+LOG_LEVEL_MAPPING = {10: "debug", 20: "info", 30: "warning", 40: "error", 50: "critical"}
+
+CONCLUSION_STATE_MAPPING = {
+    "Scheduled": TaskConclusion.UNKNOWN,
+    "Pending": TaskConclusion.UNKNOWN,
+    "Running": TaskConclusion.UNKNOWN,
```

Comment:
Do we want to add values to the `TaskConclusion` or will it not be relevant once the migration is completed?

Reply:
I think we'd remove it after the migration